src/integration-test/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/pybossa/PyBossaPlatformTest.java

```diff
@@ -42,14 +47,36 @@
             .setRatingsPerAnswer(Integer.newBuilder().setValue(5).build())
             .build();
 
-    private static PyBossaRequests requests = new PyBossaRequests(API_URL, PROJECT_ID, API_KEY);
+    private static PyBossaRequests requests;
     private static PyBossaPlatform pybossa;
 
+    @Rule
+    public ExpectedException thrown= ExpectedException.none();
+
     @BeforeClass
     public static void setUp() throws Exception {
-        deleteAllTasks(TASK_URL, API_KEY, PROJECT_ID);
-        pybossa = new PyBossaPlatform(WORKER_SERVICE_URL, WORKER_UI_URL, API_KEY,
-                API_URL, NAME, String.valueOf(PROJECT_ID), true);
+        // load config
+        File configFile = new File("src/integration-test/resources/config.yml");
+        Config config = Yaml.loadType(configFile, Config.class);
```

Comment:
I thought it might be a good idea to use a seperate config for tests. But if its not needed I can change that.
Because the config is not in the main resources, there were problems with the classloader.